src/Backend/FIPSCrypto.php

```diff
@@ -100,8 +103,11 @@ public function encrypt(
      * @throws \SodiumException
      */
     public function decrypt(
+        #[\SensitiveParameter]
         string $ciphertext,
```

Comment:
It certainly doesn't hurt to add the attribute here, but a `$ciphertext` is likely not a sensitive value.

Reply:
It isn't, but the less that leaks from a cryptography library, the better.